crates/bevy_diagnostic/Cargo.toml

```diff
@@ -10,7 +10,6 @@ keywords = ["bevy"]
 
 
 [dependencies]
-# bevy
```

Comment:
why do you remove this line?

Reply:
I don't remember doing that, added it back.